source/Mlos.Python/mlos/Spaces/HypergridAdapters/unit_tests/TestCategoricalToOneHotEncodedHypergridAdapter.py

```diff
@@ -151,7 +98,8 @@ def _test_projecting_dataframe_categorical_to_one_hot_encoding_point_from_adapte
         assert id(original_df) == id(unprojected_in_place_df)

```

Comment:
In these tests should we also check that all values in the resulting dataframe are in fact one-hot-encoded? Can we check that we have all the columns we are expecting, as well as that all the values are of the right type and within the right ranges?

Reply:
The values for the OHE dimensions/df values were already being tested.  The last commit added confirmation no CategoricalDimensions exist in the target space and that the expected dataframe columns are as expected.  This last test is a little subtle since the adapter will re-use columns if the `merge_all_categorical_dimensions` == False.